src/MassTransit/Configuration/InMemoryConfigurationExtensions.cs

```diff
@@ -7,7 +7,7 @@
     public static class InMemoryConfigurationExtensions
     {
         /// <summary>
-        /// Configure and create an in-memory bus
+        ///     Configure and create an in-memory bus
```

Comment:
yeah, I'm being pendantic, but this will annoy me to no end.

Reply:
I'm excited from your integration.